window-manager/build.gradle

```diff
@@ -22,12 +22,12 @@ plugins {
 }
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
 
     defaultConfig {
         applicationId "com.codelab.foldables.window_manager"
         minSdkVersion 23
-        targetSdkVersion 30
+        targetSdkVersion 31
```

Comment:
nit, we can also usr target `SdkVersion 30`

Reply:
Addressed in new commits. Thanks Pietro!